src/bin/cratesfyi.rs

```diff
@@ -12,12 +12,29 @@ use docs_rs::{
     Storage,
 };
 use once_cell::sync::OnceCell;
+use sentry_anyhow::capture_anyhow;
+use sentry_log::SentryLogger;
 use structopt::StructOpt;
 use strum::VariantNames;
 
 pub fn main() {
     let _ = dotenv::dotenv();
-    logger_init();
+
+    let _sentry_guard = if let Ok(sentry_dsn) = env::var("SENTRY_DSN") {
+        rustwide::logging::init_with(SentryLogger::with_dest(logger_init()));
+        Some(sentry::init((
+            sentry_dsn,
+            sentry::ClientOptions {
+                release: Some(docs_rs::BUILD_VERSION.into()),
+                attach_stacktrace: true,
+                ..Default::default()
```

Comment:
I was reading through the docs and found `environment`: https://docs.sentry.io/platforms/rust/configuration/options/#environment
> Sets the environment. This string is freeform and not set by default. A release can be associated with  more than one environment to separate them in the UI (think staging vs prod or similar).
> By default the SDK will try to read this value from the SENTRY_ENVIRONMENT environment variable (except for the browser SDK where this is not applicable).

Can you set `SENTRY_ENVIRONMENT=dev` in `.env.sample`?